helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -27,27 +27,27 @@ csiResizerImage: quay.io/k8scsi/csi-resizer
 csiResizerTag: v0.2.0
 
 # csi-driver tag
-csiDriverImage: hpe/csi-driver
-csiDriverTag: edge
+csiDriverImage: hpestorage/csi-driver
+csiDriverTag: v1.0.0-beta
 
 #csp-hpe
-cspImage: hpe/nimble-csp
-cspTag: edge
+cspImage: hpestorage/nimble-csp
+cspTag: v1.0.0-beta
 
 #parameters
-arrayIp: 10.0.0.1
+backend: 10.0.0.1
```

Comment:
I've adopted the blessed RFC1918 "example" IP address of 192.168.1.1 in most of the docs lately.

Reply:
sure..will change..